test.sh

```diff
@@ -8,7 +8,7 @@ test_bsp_server() {
   bazel run --define "maven_repo=file://$HOME/.m2/repository" //:bsp.publish
   bsp_path="$(bazel info bazel-bin)/bsp-project.jar"
   cd sample-repo
-  java -cp $bsp_path org.jetbrains.bsp.bazel.Install
+  java -cp $bsp_path org.jetbrains.bsp.bazel.install.Install
```

Comment:
This change also needs to be reflected in the Readme. Can you change that?

Reply:
I didn't change it on purpose: Readme describes installation process with coursier, but we didn't publish a new version to Maven, so that instruction is still valid.

Reply:
The instructions for coursier actually mention the snapshots repository of Maven, that Teamcity will automatically push to after a merge to master.
Perhaps you don't remember from the deep-dive we did (I wouldn't remember either, from the amount of information I dropped on you :smile: ), but there's a CI/CD running on this that we cannot see on Github